src/Humanizer.Tests.Shared/CollectionHumanizeTests.cs

```diff
@@ -87,5 +88,58 @@ public void HumanizeUsesObjectFormatterWhenSeparatorIsProvided()
             var humanized = _testCollection.Humanize(sc => string.Format("SomeObject #{0} - {1}", sc.SomeInt, sc.SomeString), "or");
             Assert.Equal("SomeObject #1 - One, SomeObject #2 - Two, or SomeObject #3 - Three", humanized);
         }
+
+        [Fact]
+        public void HumanizeHandlesNullItemsWithoutAnException()
+        {
+            new object[] { null, null }.Humanize();
```

Comment:
I don't believe it improves readability- after all, it's a single line in a method named `WithoutAnException` and we all know how the language works- if anything it's subtly misleading, but I'm happy to do whatever matches the project's style.